blog/_posts/2018-07-09-introducing-TeamBarcelonaDuo.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Introducing Team Barcelona Duo
```

Comment:
Hi @amaliacardenas ! Thanks for your pull request.  Please check the comments below.

Reply:
Hi @nerdyvaishali Thanks for your feedback. I made the corrections you suggested. Let me know if there is anything else that needs to be changed.

Reply:
Thanks @amaliacardenas ! The post looks good to me now.